src/items/equipment.js

```diff
@@ -85,4 +100,15 @@ export default class Equipment extends Item {
         super.highlightItem();
         console.log("use", this.name);
     }
+
+    // TODO: be able to have multiple different required levels for different skills
+    checkRequiredLevel() {
+        let skill = this.skill.toLowerCase();
+        let level = calcLevel(this.scene.characterData.skills[skill]);
+        if (level >= this.requiredLevel) {
```

Comment:
return (level >= this.requiredLevel);

Reply:
Already fixed here: https://github.com/CurtisGreen/rs-clicker/pull/1/files#diff-74ef585e3bac3a0d35b3f94e8d160d27c57fc7f6e685ddf8a014028eb9e122c2